docs/index.rst

```diff
@@ -10,7 +10,8 @@ Contents:
    installation
    usage
    contributing
-   authorshistory
+   authors
```

Comment:
IMO we should remove the `authors` file... it is not really necessary given the you can get that from git automatically and more accurately.